synnax/pkg/api/channel.go

```diff
@@ -37,6 +38,7 @@ type Channel struct {
 	IsIndex     bool                 `json:"is_index" msgpack:"is_index"`
 	Index       channel.Key          `json:"index" msgpack:"index"`
 	Alias       string               `json:"alias" msgpack:"alias"`
+	Internal    bool                 `json:"internal" msgpack:"alias"`
```

Comment:
The struct tags here are incorrect. Should be `json:"internal" msgpack:"internal"`